ros/src/twist_controller/twist_controller.py

```diff
@@ -1,14 +1,60 @@
-
+import rospy
+from pid import PID
+from yaw_controller import YawController
+from lowpass import LowPassFilter
 GAS_DENSITY = 2.858
-ONE_MPH = 0.44704
+FULL_BRAKE_SPEED = 0.1  # If target velocity is smaller, apply full brake
 
 
 class Controller(object):
```

Comment:
Would you be able to provide a general comment on this class ? Specifying the range and unity of each inputs really having a clear understanding of what is going on.